allauth/templates/account/email_change.html

```diff
@@ -16,14 +16,14 @@
         {% slot body %}
             {% csrf_token %}
             {% if current_emailaddress %}
-                {% element field disabled=True type="email" value=current_emailaddress.email %}
```

Comment:
I don't think these (id=) changes belong here as part of this PR?

Reply:
These changes were intentional to address the empty `for` attributes mentioned in the PR description.  See [this line from field.html](https://github.com/pennersr/django-allauth/blob/ef0f66b8131964941f9b1dbe93086aa509602383/allauth/templates/allauth/elements/field.html#L3).  If you don't include an `id`, then the `label` value will be empty and not point to any input tag, which is invalid HTML.

Is there a better way to solve this?  I see `id` attributes used [for other field elements](https://github.com/pennersr/django-allauth/blob/ef0f66b8131964941f9b1dbe93086aa509602383/allauth/templates/mfa/recovery_codes/index.html#L11).

Reply:
Sorry, it's not clear to me how I could leverage Django's `auto_id` since that's associated with a Django form and form field, and `{% element field %}` is sometimes used for fields that aren't part of a Django form or even have a `name` attribute (like this disabled one).  Or are you suggesting to change the `field.html` template to default to something like `id_%s` like a Django form does if an explicit `id` is not provided?  That could get a little tricky too since `name` is not required, although we could add it.

Reply:
The `fields.html` template over at the example project already uses `auto_id` as the default by using:

`id=bound_field.auto_id`

So, indeed, for fields that do not originate from a form field we can populate an `id` ourselves like you are doing. Though there is one field on line which could use `auto_id` I think.

Reply:
Thanks for the pointer to the example project.  I think we're on the same page now.  I just changed the one email field backed by a form to use `auto_id`.